docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -8,25 +8,30 @@ Introduction
+In principle, one obtains a vibrational spectrum which can be quite difficult to analyse; in crystalline materials this
+related to the wavevector-dependent *phonon* spectrum.
+In order to better understand experimental outputs, one can compare with results from modelling.
+*Ab initio* calculations, especially within density-functional theory (DFT) [#Kohn1964]_, have proven quite successful in predicting vibrational spectra.
 
 .. image:: ../images/dft_phonon_scheme.png
     :align: center
 
-Usual approach in calculating phonon spectra within DFT is presented in the figure above. First, one defines initial
-guess for the structure in interest. The initial guess should be as close as possible to an experimental structure.
-Then, one optimise structure within DFT. By optimising structure one should understand finding the closest to the
-initial guess local minimum of energy within accuracy of DFT method. At the closest minimum there
-should be no effective force on atoms. For that structure the dynamical matrix, e.g, second derivative of system Hamiltonian with respect to
-atomic displacements is calculated. Eigenvectors obtained from diagonalisation of this matrix are atomic displacements
-and eigenvalues are squared frequencies of vibrations. Those vibrations are sometime called *modes* or *fundamentals*.
-Using atomic displacements and frequencies obtained by DFT method one can calculate theoretical :math:`S(\mathbf{Q}, \omega)`. :math:`S(\mathbf{Q}, \omega)`
-is calculated for each atom separately. Total S is sum over all partial atomic contributions.
+The usual workfow for calculating phonon spectra within DFT is presented in the figure above. First, one defines an
+initial guess for the structure in interest.
+The initial guess should be as close as possible to an experimental structure, and is usually derived from elastic X-ray and/or neutron scattering measurements.
+Then the structure parameters are locally optimised within DFT, finding the nearest structure that minimises the DFT energy.
+At this point, there should be no effective force on the atoms.
+For this "relaxed" structure the dynamical matrix is calculated, either by finite displacements or perturbation theory.
+The dynamical matrix is related to the Hessian (the second derivative of system Hamiltonian with respect to atomic displacements) by a Fourier transform;
+the eigenvectors obtained from diagonalisation of this matrix are atomic displacements
+and the eigenvalues are squared frequencies of vibrations.
+These vibrational *modes* are related to the *fundamental* vibrational excitations of the system,
+and using this displacement and frequency information one can calculate theoretical :math:`S(\mathbf{Q}, \omega)`.
+In Abins :math:`S(\mathbf{Q}, \omega)` is calculated for each atom separately.
+The total S is then obtained as a sum over all partial atomic contributions.
```

Comment:
math font

Reply:
Or kill the repetition and give more context:
```rst
In Abins :math:`S(\mathbf{Q}, \omega)` is calculated for each atom separately;
the total spectrum is then obtained as a sum over all partial atomic contributions.
```